datafusion/expr/src/logical_plan/builder.rs

```diff
@@ -351,7 +351,9 @@ impl LogicalPlanBuilder {
                 schema: _,
             }) if missing_cols
                 .iter()
-                .all(|c| input.schema().field_from_column(c).is_ok()) =>
+                // TODO: how to return err in closure, maybe there is hethod like try_all in iter?
```

Comment:
the whole point if the `is_ok()` call is to ignore the error (aka the error should not be returned)

Reply:
yes, I agree with you